frontend/config/mechs.ts

```diff
@@ -24,14 +25,18 @@ export const MECHS: Mechs = {
       name: 'Agent Mech',
       contract: new MulticallContract(
         '0x77af31De935740567Cf4fF1986D04B2c964A786a',
-        AGENT_MECH_ABI,
+        AGENT_MECH_ABI.filter(
```

Comment:
probably better to put on the ABI itself, but this is fine for now; no need for JsonFragment either, can use Abi from types